lib/rubocop/config_loader.rb

```diff
@@ -195,9 +195,8 @@ def dirs_to_search(target_dir)
       end
 
       def old_auto_config_file_warning
-        raise RuboCop::Error,
-              'rubocop-todo.yml is obsolete; it must be called' \
-              " #{AUTO_GENERATED_FILE} instead"
+        $stderr.puts 'rubocop-todo.yml is obsolete; it must be called' \
```

Comment:
[warn](http://ruby-doc.org/core-2.2.3/Kernel.html#method-i-warn) might be a better choice here.